xtask/src/glob.rs

```diff
@@ -0,0 +1,35 @@
+use std::path::Path;
+
+use anyhow::Context;
+use glob::glob;
+
+use crate::result::LogIfError;
+
+pub(crate) fn visit_files(
+    path: impl AsRef<Path>,
+    glob_expr: &str,
+    found_err: &mut bool,
```

Comment:
I feel like we could do something else here where we'd avoid having this out param.

And/or have it be a `Status` enum instead to make things clearer (especially for closures).

Reply:
Hmm…yeah, this isn't a great API right now. Changed the `bool` to be a more explicit `enum ErrorStatus` with d5db0e13, and I think that that's a good change, overall. I experimented with trying to return things by value in 224da534, but I'm not persuaded that it's an improvement overall without more work.